### PR TITLE
busybox: enable testcase logging

### PR DESCRIPTION
**Pull request by mateusz-bloch**
JIRA: CI-384	

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [x] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: busybox/19-enable-testcase-logging.patch [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
++++ _phoenix/busybox-1.27.2/testsuite/testing.sh	2023-10-27 17:17:16.635573994 +0200
+@@ -87,7 +87,7 @@
+ 
+   $ECHO -ne "$3" > expected
+   $ECHO -ne "$4" > input
+-  [ -z "$VERBOSE" ] || echo ======================
++  echo "Running testcase $NAME"
+   [ -z "$VERBOSE" ] || echo "echo -ne '$4' >input"
+   [ -z "$VERBOSE" ] || echo "echo -ne '$5' | $2"
+   $ECHO -ne "$5" | eval "$2" > actual
```
Comment (damianloew) [inline busybox/19-enable-testcase-logging.patch:23]: It seems that there are also other places where test cases start:
Screenshot from 2023-11-08 12-35-25

There you also don't know when you should start parsing output.

Please correct it, run all the tests and make sure that all the output is formatted correctly.

**Review by damianloew — CHANGES_REQUESTED** [review]

File: busybox/19-enable-testcase-logging.patch (hunk 1 above)
Comment (mateusz-bloch) [reply busybox/19-enable-testcase-logging.patch:23]: image

Comment (damianloew) [reply busybox/19-enable-testcase-logging.patch:23]: Thanks, one more nitpick: maybe it would be good to do  ---Testcase 'testcase name' starting---

Comment (mateusz-bloch) [reply busybox/19-enable-testcase-logging.patch:23]: Did you mean that? 
image

Comment (damianloew) [reply busybox/19-enable-testcase-logging.patch:23]: Isn't it the same?

Comment (mateusz-bloch) [reply busybox/19-enable-testcase-logging.patch:23]: Is that what it was about?: 
image

I didn't understand, previously I changed only that the testcase was printed and not the entire path in one case of printning logs.

Comment (damianloew) [reply busybox/19-enable-testcase-logging.patch:23]: Yeah, exactly

**Review by damianloew — APPROVED** [review]

**Pull request merged**